wrims_v2/wrims_v2/src/test/test_heclib7/Test_heclib7.java

```diff
@@ -95,9 +117,25 @@ public void writeData(String dvPath){
 			}
 		}catch (Exception e) {
 			System.out.println("Could not open dv file. "+e);
-			return;
+			return null;
 		}
+		return outPath;
 	}
 	
+	public TimeSeriesContainer readData(String dvPath, String fullName){
+		HecDss dvDss;
+		DataContainer tsc;
+		try{
+			dvDss = HecDss.open(dvPath);
+			tsc = dvDss.get(fullName);
```

Comment:
While I don't have a "solution" to the bug. I found a workaround: this `get(String)` method has an overload `get(String pathname, boolean readEntireSet)`, the `get(String pathname)` that is being used here calls into the overload as `get(pathname, false)` so the entire set doesn't get read. 

When I add `true` instead, I get back all the expected values.